examples/verify-common.sh

```diff
@@ -72,7 +73,7 @@ cleanup () {
 
 _curl () {
     local arg curl_command
-    curl_command=(curl -s)
+    curl_command=(curl -s "${EXTRA_CURL_ARGS[@]}")
```

Comment:
this isnt necessary - it already handles additional args - see here https://github.com/envoyproxy/envoy/blob/master/examples/DEVELOPER.md#utility-functions-responds_with